### DISCUSSION TITLE
Difference between stabilizationWindowSeconds and cooldownPeriod settings

### DISCUSSION THREAD

**Discussion by Alekky09** [kedacore/keda#4850] [Q&A / Need Help]
I've looked in the docs but couldn't find anything for the `stabilizationWindowSeconds` param.
cooldownPeriod is explained as:
```The period to wait after the last trigger reported active before scaling the resource back to 0. By default, it’s 5 minutes (300 seconds)```

This seems very close if not the same as what `stabilizationWindowSeconds` should do too.
Could someone please tell me the gist of the difference?

I've been looking into these cause I'm trying to scale down my pods faster.

**Comment by JorTurFer** [comment]
Hi
They do almost the same but at different places:
- cooldownPeriod is the time before the operator will scale to 0 after the scaledobject has been deactivated (because all the triggers return 0)
- stabilizationWindowSeconds is the that the HPA controller will wait after scaling in to scale in again

Both are the period to wait before scaling it, but one is used by the operator and the other by the HPA controller
